Week2/Homework/03_recipecard.js

```diff
@@ -0,0 +1,25 @@
+/*Ever wondered how to make a certain meal? Let's create a recipe list with JavaScript!
+
+Declare a variable that holds an object (your meal recipe).
+Give the object 3 properties: a title (string), a servings (number) and an ingredients (array of strings) property.
+Log each property out seperately, using a loop (for, while or do/while)
+It should look similar to this:
+
+Meal name: Omelete
+Serves: 2
+Ingredients:
+4 eggs
+2 strips of bacon
+1 tsp salt/pepper*/
+
+"use strict";
+
+let recipe = {
+  "Meal name": "Pizza",
+  Serves: 4,
+  Ingredients: ["cheese", "flour", "pepperoni"]
+};
+console.log(recipe);
+for (var key in recipe) {
+  console.log(`${key} : ${recipe[key]}`);
+}
```

Comment:
👍 but a few things.
1. Should use `const` instead of `let` here since we don't reassign `recipe` object anywhere.
2. As a good practice, you should try to use double quotes for all properties of the objects if you are using them at all i.e have keys`Serves` and `Ingredients` in double quotes too.
3. I would suggest you also look out for a method named `Object.keys` which is a nice new alternative to `for-in` loop